multi_folder_download.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import os
+import subprocess
+import sys
+import time
+from datetime import datetime
+from distutils.util import strtobool
+
+import django
+from dateutil.relativedelta import relativedelta
+from dateutil.tz import tzutc
+from django.contrib.gis.geos import GEOSGeometry
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'geospaas_processing.settings')
+
+import geospaas_processing.downloaders as downloaders
+from geospaas.catalog.models import Dataset
+
+
+def main_function(ar):
```

Comment:
Change name.

Reply:
Change name of which one? `main_function` or `ar`. I name it as ar since it is repeated many times in the function and it is better not to slipt the lines several times!

If you mean main_funciton, I think it is a good name for it because this is the main function that is called.

Reply:
When writing script which are going to be called from the command line, please keep to this simple convention:
```python
def main():
    print("Hello World!")  # the code goes here

if __name__ == "__main__":
    main()
```
See https://realpython.com/python-main-function/ for more details.

Reply:
Done in the next commit.

Reply:
DOne in the next commit.